docs/design/autoware-architecture/sensing/index.md

```diff
@@ -1,5 +1,34 @@
-# Sensing component design
+# Sensing stack design
 
-!!! warning
+## Overview
 
-    Under Construction
+Sensing stack is a collection of modules that apply some primitive pre-processing to the raw sensor data.
+
+The sensor input formats are defined in this stack too.
+
```

Comment:
Almost same commnet : https://github.com/autowarefoundation/autoware-documentation/pull/180/files#r1011824912
```suggestion
## Role
- Abstraction of data format for each sensor to absorb vendor-specific differences
- Perform common sensor data processing required by each component
```
I am not confident in my English grammar.

Reply:
Added `##Role` section with some modifications from your suggestions in https://github.com/autowarefoundation/autoware-documentation/pull/180/commits/a56bd64a9e1062d7905143dc4c83200dddddcca3